how-to/how-to-use-k8s-with-cri-containerd-and-kata.md

```diff
@@ -69,28 +87,33 @@ $ cat << EOT | sudo tee /etc/containerd/config.toml
 EOT
 ```
 
-### Configure Kubelet to use containerd
+**Note:** Unless configured otherwise, the default runtime is set to `runc`.
 
-In order to allow kubelet use containerd (using CRI interface) configure the service to
-point to containerd socket.
+## Configure Kubelet to use containerd
 
-```bash
-# Configure k8s to use containerd
-$ sudo mkdir -p  /etc/systemd/system/kubelet.service.d/
-$ cat << EOF | sudo tee  /etc/systemd/system/kubelet.service.d/0-containerd.conf
-[Service]                                                 
-Environment="KUBELET_EXTRA_ARGS=--container-runtime=remote --runtime-request-timeout=15m --container-runtime-endpoint=unix:///run/containerd/containerd.sock"
-EOF
-$ sudo systemctl daemon-reload
-```
+In order to allow kubelet to use containerd (using the CRI interface), configure the service to point to the `containerd` socket.
+
+- Configure Kubernetes to use `containerd`
+
+  ```bash
+  $ sudo mkdir -p  /etc/systemd/system/kubelet.service.d/
+  $ cat << EOF | sudo tee  /etc/systemd/system/kubelet.service.d/0-containerd.conf
+  [Service]                                                 
+  Environment="KUBELET_EXTRA_ARGS=--container-runtime=remote --runtime-request-timeout=15m --container-runtime-endpoint=unix:///run/containerd/containerd.sock"
+  EOF
+  ```
+
+- Inform systemd about the new configuration
+
+  ```bash
+  $ sudo systemctl daemon-reload
+  ```
 
-### Optional: Configure proxy
+## Configure proxy - OPTIONAL
 
-If you are behind a proxy this script will configure your proxy for docker
-kubelet and containerd.
+If you are behind a proxy, use the following script to configure your proxy for docker, kubelet, and containerd:
 
 ```bash
-# Set proxys
```

Comment:
I wonder if coding this as a `#!/bin/bash` script here will make it easier for a cut/paste for the user. I know we don't tend to do that for other inline scripts, so am happy to leave as is as well.

Reply:
Good point - I've simplified that block to not require bash now.